apps/voting-disputable/app/src/script.js

```diff
@@ -425,6 +426,18 @@ function marshallVote({
     startDate: marshallDate(startDate),
   }
 }
+function marshallDisputableInfo({ actionId, pauseDuration, pausedAt, status }) {
+  return {
+    actionId,
+    pauseDuration,
+    pausedAt: marshallDate(pausedAt),
+    status,
+  }
+}
+actionId: '2'
+pauseDuration: '0'
+pausedAt: '1594081479'
+status: '1'
```

Comment:
Should these be here?

Reply:
haahah no!! i dont know  how it got there. thanks!